File: frontend/src/dto/input/CreateChatbotTemplateDTO.ts
export interface CreateChatbotTemplateDTO {
  chatbotName: string;
  description: string;
  chatbotModel: string;
  chatbotIcon: string;
  chatbotLanguage: string;
  chatbotRole: string;
  chatbotTone: string;
  welcomeMessage: string;
  workspaceId: string;
}

File: backend/src/main/java/ch/uzh/ifi/imrg/platform/rest/dto/input/CreateChatbotTemplateDTO.java
package ch.uzh.ifi.imrg.platform.rest.dto.input;

import lombok.Data;

@Data
public class CreateChatbotTemplateDTO {
  private String chatbotName;
  private String chatbotIcon;
  private String chatbotRole;
  private String chatbotTone;
  private String welcomeMessage;
  private Boolean isActive;
}
